src/Pages/Homepage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Container, Row, Col } from "react-bootstrap";
+import Header from '../Frame/Header/Header';
+import FreeWeekRotation from '../Components/FreeWeekRotation/FreeWeekRotation';
+import SearchField from '../Components/SearchField/SearchField';
+import { fetchChampionList } from '../Modules/APIGateway';
+
+class Homepage extends React.Component {
+    state = {
+        champions: null,
+    }
+
+    async componentDidMount() {
+        // Get all champions from the server when app starts
```

Comment:
Let's integrate redux into the app!
https://redux.js.org/
Please move the fetch to an action creator, and store the champion list in a redux store.
Call the action on DidMount and use the data from the store :) add an isLoading prop so we can display a nice spinner